marlowe-dashboard-client/src/Wallet/Types.purs

```diff
@@ -0,0 +1,41 @@
+module Wallet.Types
+  ( WalletLibrary
+  , WalletNicknameKey
+  , WalletDetails(..)
+  , PubKeyHash
+  ) where
+
+import Prelude
+import Data.Generic.Rep (class Generic)
+import Data.Map (Map)
+import Data.Newtype (class Newtype)
+import Data.Tuple (Tuple)
+import Foreign.Generic (class Decode, class Encode, defaultOptions, genericDecode, genericEncode)
+
+type WalletLibrary
+  = Map WalletNicknameKey WalletDetails
+
+-- N.B. make the nickname (string) the first key so we get alphabetical ordering for free
```

Comment:
what does N.B. stand for?

Reply:
"nota bene". Sorry, old habit. I'll change it to "note".

Reply:
I never knew what it stood for but always knew what it meant! It's pretty common in written British English.